0.data-download/scripts/data_loader.py

```diff
@@ -12,69 +12,78 @@ def load_data(data_directory, adult_or_pediatric="all"):
 
     # Define data paths
     data_directory = "../0.data-download/data/"
-    sample_info_file = pathlib.Path(
+    model_file = pathlib.Path(
         data_directory, "Model_age_column_cleaned.csv"
     )
     dependency_data_file = pathlib.Path(data_directory, "CRISPRGeneDependency.csv")
 
     # Load data
     dependency_df = (
-        pd.read_csv(dependency_data_file, index_col=0).reset_index().dropna(1)
+        pd.read_csv(dependency_data_file, index_col=0).reset_index().dropna(axis=1)
     )
-    sample_df = pd.read_csv(sample_info_file, index_col=0)
+    model_df = pd.read_csv(model_file, index_col=0)
 
-    # rearrange sample info and gene dependency dataframe indices so DepMap_IDs are in alphabetical order
-    sample_df = sample_df.sort_index(ascending=True)
-    sample_df = sample_df.reset_index()
+    # rearrange model info and gene dependency dataframe indices so DepMap_IDs are in alphabetical order
+    model_df = model_df.sort_index(ascending=True)
+    model_df = model_df.reset_index()
 
     dependency_df = (
         dependency_df.set_index("DepMap_ID").sort_index(ascending=True).reset_index()
     )
 
-    # searching for similar IDs FROM dependency df IN sample df
+    # searching for similar IDs FROM dependency df IN model df
     dep_ids = dependency_df["DepMap_ID"].tolist()
-    dep_vs_samp_ids = set(dep_ids) & set(sample_df["DepMap_ID"].tolist())
+    dep_vs_mod_ids = set(dep_ids) & set(model_df["DepMap_ID"].tolist())
 
-    # searching for similar IDs FROM sample df In dependency df
-    samp_ids = sample_df["DepMap_ID"].tolist()
-    samp_vs_dep_ids = set(samp_ids) & set(sample_df["DepMap_ID"].tolist())
+    # searching for similar IDs FROM model df In dependency df
+    mod_ids = model_df["DepMap_ID"].tolist()
+    mod_vs_dep_ids = set(mod_ids) & set(model_df["DepMap_ID"].tolist())
 
-    # subset data to only matching IDs (samples in both dependency and sample data)
-    sample_df = sample_df.loc[sample_df["DepMap_ID"].isin(dep_vs_samp_ids)].reset_index(
+    # subset data to only matching IDs (samples in both dependency and model data)
+    model_df = model_df.loc[model_df["DepMap_ID"].isin(dep_vs_mod_ids)].reset_index(
         drop=True
     )
-    dependency_df = dependency_df.loc[dependency_df["DepMap_ID"].isin(samp_vs_dep_ids)]
+    dependency_df = dependency_df.loc[dependency_df["DepMap_ID"].isin(mod_vs_dep_ids)]
 
     if adult_or_pediatric != "all":
-        sample_df = sample_df.query(
+        model_df = model_df.query(
             "age_categories == @adult_or_pediatric"
         ).reset_index(drop=True)
-        samples_to_keep = sample_df.reset_index(drop=True).DepMap_ID.tolist()
+        model_to_keep = model_df.reset_index(drop=True).DepMap_ID.tolist()
         dependency_df = dependency_df.query(
             "DepMap_ID == @samples_to_keep"
         ).reset_index(drop=True)
 
-    return sample_df, dependency_df
+    return model_df, dependency_df
```

Comment:
Please sort the returns. There is no guarantee that the models are in the same order, which is important if you're concatenating downstream (and general good practice to confirm consistency between two dataframes that may be expected to be equivalent in the sample dimension.

Reply:
It looks like there is already sorting happening on these lines:
![image](https://user-images.githubusercontent.com/116824883/214733886-fed98512-708a-4005-a1b9-2157fbe0ea92.png)
Do I need to sort again?

Reply:
re-reading this now, I think I meant "reindex" rather than "sort".

Returning reindexed dataframes according to the common samples will ensure that they are ordered properly. See https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.reindex.html for more details!

Reply:
Ok, in the 2.test-tain_split.ipynb file where these returns are used (ln [4]), there is a verification performed to ensure they are aligned and when I add the reindex here it conflicts. I'd like to go over this with you next time we meet to resolve this!